### DISCUSSION TITLE
委托（刷怪塔为何不刷怪）

### DISCUSSION THREAD

**Discussion by LT-UT** [share121/inter-knot#3384] [其他]
![Snipaste_2024-08-21_13-36-26](https://github.com/user-attachments/assets/0578b094-badb-4ede-820b-d3caeac67783)
尝试解决电子阳痿、
Java版
里面很黑，也放了水，就是不见怪
这是为何？

**Comment by Lainisang** [comment]
新版本MC的刷刷怪策略好像改了，建议上b站

**Comment by jsishsjsj** [comment]
有可能是离得太近，远离128格看看。
另外这种刷怪塔的效率并不高，建议更换<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by LT-UT** [reply]
正在改呢，不知到行不行

**Reply by isle200inf** [reply]
128格之外就不刷辣，刷怪范围是24格-128格

**Comment by jiangshixie** [comment]
之前我也这么做刷怪塔，等了半个小时也刷不出来怪<img src="https://bbs-static.miyoushe.com/static/2024/03/28/748a0ed5759178eadf0db99ddd5609bf_8473130628389224205.png" alt="黄泉 迷茫" width="75" >

**Reply by LT-UT** [reply]
水大概放了多少？

**Reply by jiangshixie** [reply]
很久之前的事了，放了多少不清楚

**Reply by LT-UT** [reply]
我这个塔就是水放太多而刷不出来

**Comment by liubanlaobanzhang** [comment]
我记得是刷怪机制改了？<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by LT-UT** [reply]
我这个是水放太多而刷不出来

**Reply by LT-UT** [reply]
问一下末影人，蜘蛛冲不下去咋解决

**Comment by Alsuhold** [comment]
高版本建议做专项刷怪塔，不同的怪在高版本也会相互干扰

**Reply by LT-UT** [reply]
干不动了<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Comment by SepeiJohn** [comment]
如果是寻路式刷怪塔的话需要放活板门什么的，不然怪不会自己走下去

**Reply by LT-UT** [reply]
活扳门放哪？

**Reply by SepeiJohn** [reply]
原理是被打开的活板门会被怪检测为完整方块，然后怪就会自己走上去，而因为活板门是打开的所以怪会掉下去

**Reply by LT-UT** [reply]
我建的是怪物被水利留冲后摔死的那种

**Comment by isle200inf** [comment]
看看内部什么样，怪不能生成在水里

**Reply by LT-UT** [reply]
改了，加了个刷怪空间，在刷了，现在在优化

**Comment by dinkaka** [comment]
看你这个形状，你给怪留了空间了吗？
而且刷怪范围是玩家周围半径128的圆形，你要到它上空+128的地方才能保证只有刷怪塔里面刷怪，不然全刷到地下了

**Comment by QuasarRyan** [comment]
经典错误，详见[如何制作百万刷怪塔](https://www.bilibili.com/video/BV1tg4y1U7qy)。

**Comment by Despair-Y** [comment]
建在海上啊，陆地上全是实体方块，实体方块会刷怪的啊，建海上就只有刷怪塔是实体方块，所以建海上效率才高，如果你弄个空置域我也没意见（doge）
